src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorage/AzureTableStorageSink.cs

```diff
@@ -39,20 +38,6 @@ public class AzureTableStorageSink : ILogEventSink
         /// <param name="storageAccount">The Cloud Storage Account to use to insert the log entries to.</param>
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
         /// <param name="storageTableName">Table name that log entries will be written to. Note: Optional, setting this may impact performance</param>
-        public AzureTableStorageSink(
```

Comment:
Sounds good! Definitely seems like there are enough changes on dev to bump the major.